packages/fuel-indexer-tests/src/fixtures.rs

```diff
@@ -93,7 +99,7 @@ pub async fn setup_test_fuel_node(
         Some(_) => {
             EnvFilter::try_from_default_env().expect("Invalid `RUST_LOG` provided")
         }
-        None => EnvFilter::new("info"),
+        None => EnvFilter::new("println"),
```

Comment:
Is this supposed to be "println"? This will default to the `ERROR` level if it's invalid.